### ISSUE TITLE
the steps for building ONL by hand  need to be updated

### ISSUE THREAD

**Issue by mx-zheng** [opencomputeproject/OpenNetworkLinux#753]
now the steps for  building ONL by hand  is too old :

![image](https://user-images.githubusercontent.com/22835075/88478563-1d23c800-cf7c-11ea-96c8-b7a219b5cfc0.png)

root@ser:/home/OpenNetworkLinux# source setup.env
bash: setup.env: No such file or directory

**Comment by laiguoliang-lyr** [comment]
docker/tools/onlbuilder --8

**Comment by Yi-Tseng** [comment]
Try `make docker` or `make docker-debug` if you want to run commands manually

**Comment by nosignal** [comment]
`docker/tools/onlbuilder (-8)` means that if you want to specify a version of builder image, add -8(or some other version you wanted). By default, the version is 8.
Please try `docker/tools/onlbuilder --help` to see more usages. This step is never an issue.

**Comment by paulmenzel** [comment]
Yes, `(-8)` just denotes it’s an optional argument. Please close this issue.
